src/common/__tests__/base-store.test.ts

```diff
@@ -105,10 +94,17 @@ describe("BaseStore", () => {
   let store: TestStore;
 
   beforeEach(async () => {
+    const dis = getDisForUnitTesting({ doGeneralOverrides: true });
```

Comment:
Most of times `doGeneralOverrides: true` is the correct way. Because it override e.g. `electron` from unit tests. 
I would suggest that we keep it there for a while and make it default in separate PR if we really want to do so in future. 

It's there to make sure that you as a developer know that it's doing the overrides. Less magic.